src/modules/MessageSearch/context/hooks/useSetChannel.ts

```diff
@@ -1,16 +1,16 @@
-import type { SendbirdGroupChat } from '@sendbird/chat/groupChannel';
 import { useEffect } from 'react';
 import type { Logger } from '../../../../lib/SendbirdState';
 import * as messageActionTypes from '../dux/actionTypes';
+import { SdkStore } from '../../../../lib/types';
 
 interface MainProps {
   channelUrl: string;
   sdkInit: boolean;
 }
 interface ToolProps {
-  sdk: SendbirdGroupChat;
+  sdk: SdkStore['sdk'];
   logger: Logger;
-  messageSearchDispatcher: ({ type: string, payload: any }) => void;
+  messageSearchDispatcher: (param: { type: string, payload: any }) => void;
```

Comment:
```suggestion
  messageSearchDispatcher: (props: { type: string, payload: any }) => void;
```

Reply:
In a typical development environment, we might use 'props,' but in the context of developing with React, 'props' feels like a reserved word primarily used within components. Therefore, I've used 'params